<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comment_user</class>
 <widget class="QWidget" name="comment_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>评论人</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_comment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>评论</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
